tools/generate-docs.sh

```diff
@@ -0,0 +1,25 @@
+#!/usr/bin/env bash
+
+# Generate docs!
+go run github.com/hashicorp/terraform-plugin-docs/cmd/tfplugindocs
+
+# Generate subcategories
+# Follow https://github.com/hashicorp/terraform-plugin-docs/issues/156 to see if anything is merged upstream. If so, we should use that.
+SCRIPTPATH="$( cd -- "$(dirname "$0")" >/dev/null 2>&1 ; pwd -P )"
+subcategories="$(cat ${SCRIPTPATH}/subcategories.json)"
+
+echo -e "\nAdding subcategories to docs..."
+for f in $(find ./docs/ -name "*.md"); do
+    f=${f#"./docs/"}
+    f="${f%.*}"
```

Comment:
How about this?
```suggestion
for f in $(find ./docs/ -name "*.md" | sed -e 's|^\./docs/||' -e 's|\.md$||'); do
```

I found Bash regex substitutions a bit hard to grasp. Feel free to ignore this.